Cargo.toml

```diff
@@ -1,30 +1,22 @@
 [package]
-name = "tsp"
-version = "0.2.3"
-authors = ["Adrian Brink <adrian@brink-holdings.com>"]
+name = "abci"
+version = "0.3.0"
+authors = ["Adrian Brink <adrian@brink-holdings.com>", "Jackson Lewis <st.japa6@gmail.com>"]
 license = "MIT/Apache-2.0"
-description = "An implementation of the TSP protocol used by Tendermint Core. Allows the building of COSMOS apps in Rust."
+description = "A ABCI server for Rust. Compatible with Tendermint 0.22."
 homepage = "https://tendermint.com/docs/guides/app-development"
-repository = "https://github.com/tendermint/rust-tsp"
-documentation = "https://docs.rs/tsp/0.2.3/tsp/"
-keywords = ["tsp", "tendermint", "cosmos", "blockchain", "proof-of-stake"]
-categories = ["encoding"]
+repository = "https://github.com/jacksoncoder/abci"
```

Comment:
?? 404

Reply:
Yeah, this should be: https://github.com/tendermint/rust-tsp I guess.

Reply:
Yeah whoops, forgot to change that.